### ISSUE TITLE
Ensure new ways of invoking machine translations conform to permission settings 

### ISSUE THREAD

**Issue by charludo** [digitalfabrik/integreat-cms#2063]
### Motivation

Once they are merged, we want to make sure the new ways of invoking machine translations that will be introduced in #1923 and #2062 also conform to - and visually reflect in the UI - the permission settings that will be enforced from #2061 forward.  

### Proposed Solution

In the `get` methods of the respective views, check if MT is allowed for the current content type / user / target language combination. If not, do not show the corresponding UI elements.

### Alternatives

Disable them instead and show a hint when hovering.

### Additional Context

#2061 will already make sure no forbidden translation will be executed, but it would be better to stop the user from trying and having to see an error message in the first place.

### Design Requirements

Decide on exact solution

### Blocked by:
- [ ] #1875 
- [ ] #1459 
- [x] #1923

**Comment by timoludwig** [comment]
Hmm, I'm not sure about this - in my opinion, splitting features from permission checks was a one-time exception for #1831 and #1927. However, I would not recommend this pattern for our general issue management and would stop this after merging #2061... yes, having smaller sub-tasks is generally preferable, but permission checks should always come with the issue they are related to. Otherwise, we might accidentally allow certain functions to users for whom they are not intended. So in my opinion, we should add this task to the issue descriptions of #1459, #1875 and #1923 instead of implementing the checks afterwards.

**Comment by timoludwig** [comment]
@charludo Did I understand correctly that you implemented the permission checks as part of the PR #2062 now? Then this ticket can be closed, right?

**Comment by charludo** [comment]
Yes, since #1923 isn't / won't be worked on anymore, this issue is no longer needed. Feel free to close it :)
